iOS/PixelDefinitions/params_dictionary.json5

```diff
@@ -53,70 +53,64 @@
         "description": "Underlying NSError code domain",
         "examples": ["NSURLErrorDomain"]
     },
-    "widePixelAppName": {
+    "wideEventAppName": {
         "key": "app.name",
         "type": "string",
         "description": "The name of the app sending the pixel",
         "enum": ["DuckDuckGo iOS"]
     },
-    "widePixelAppVersion": {
+    "wideEventAppVersion": {
         "key": "app.version",
         "type": "string",
         "description": "The version of the app sending the pixel"
     },
-    "widePixelInternalUser": {
```

Comment:
This is no longer a common param, you can include it in events if you need it but we should only do that when it really makes sense.